s3/pkg/meta/db/drivers/tidb/bucket.go

```diff
@@ -640,11 +646,11 @@ func (t *TidbClient) GetBucketVersioning(ctx context.Context, bucketName string)
 	return
 }
 
-func (t *TidbClient) CreateBucketSSE(ctx context.Context, bucketName string, sseType string) error {
+func (t *TidbClient) CreateBucketSSE(ctx context.Context, bucketName string, sseType string, sseKey []byte) error {
 	log.Infof("create bucket[%s] SSE info[%s] into tidb ...\n", bucketName, sseType)
 
-	sql := "insert into bucket_sseopts(bucketname, sse) values(?,?);"
```

Comment:
Hi @rhsakarpos Can we save SSE settings in buckets table instead of bucket_sseopts?

Reply:
will do @sfzeng, i believe RC2 was yesterday or is today, in which case can this be done after the RC2 cut ?

Reply:
OK, how about creating an issue to trace it?

Reply:
here goes #792